components/Layout/AudioControls.vue

```diff
@@ -123,7 +123,8 @@ export default Vue.extend({
       'setNextTrack',
       'setPreviousTrack',
       'unpause',
-      'pause'
+      'pause',
```

Comment:
I think a few of those could be removed

Reply:
Ah good catch, I've removed 'pause' and 'unpause' where they're not needed